package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itwin/reality-data-client",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
version management, managed by the beachball changes

Reply:
do you mean I must not change this line?

Reply:
correct; take a look at:
https://github.com/iTwin/reality-data-client/blob/master/CONTRIBUTING.md#developer-notes (last bullet point)
https://github.com/microsoft/beachball